tests/Unit/Http/Controllers/RequestControllerTest.php

```diff
@@ -80,8 +94,17 @@ public function testCreateFunction()
     public function testStoreFunction()
     {
         $this->faker = Faker::create();
+        $this->notification = Notification::fake();
```

Comment:
thử dùng mockery ddi em

Reply:
em thử dùng rồi ạ

Reply:
em không dùng được mockery ạ. Em đã tìm thử đọc trong doc Laravel với Google search thì em không thấy ai dùng mockery để test Notification. Em toàn thấy dùng Notification fake thôi ạ